src/modules/land_detector/LandDetector.cpp

```diff
@@ -57,6 +57,16 @@ LandDetector::LandDetector() :
 	ScheduledWorkItem(px4::wq_configurations::hp_default),
 	_cycle_perf(perf_alloc(PC_ELAPSED, "land_detector_cycle"))
 {
+	_landDetected.timestamp = hrt_absolute_time();
+	_landDetected.freefall = false;
+	_landDetected.landed = true;
+	_landDetected.ground_contact = false;
+	_landDetected.maybe_landed = false;
```

Comment:
Is it possible to default initialize these values at the struct definition instead?